lesson_1.1/src/AddTodoForm.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import TodoList from './TodoList';
+function AddTodoForm() {
```

Comment:
I'd add a blank line between import statements and function to give more definition to the readability.